buildscript/src/main/kotlin/Dependencies.kt

```diff
@@ -60,8 +60,7 @@ object Dependencies {
     // 1.6.x <-> AGP 3.6.x
     // 2.0.x <-> AGP 4.0.x
     // 2.1.x <-> AGP 4.1.x
-    // TODO: MBS-9089 2.0.94
-    val r8 = "com.android.tools:r8:1.6.89"
+    val r8 = "com.android.tools:r8:2.0.94"
```

Comment:
Had to revert #530. The problem is relevant to IDEA 2020.2. In AS 4.x it works fine. 
From this point, see no reason to use an unsupported r8 version.